platform-grpc-service-framework/src/main/java/org/hypertrace/core/serviceframework/grpc/GrpcPlatformServiceContainer.java

```diff
@@ -199,6 +199,23 @@ public boolean healthCheck() {
     }
   }
 
+  private Duration[] generateLatencyHistogramBuckets() {
+    ArrayList<Duration> histogram_buckets = new ArrayList<>();
+    int factor = 3;
+    long curr_duration_ms =
+        getAppConfig().hasPath(MAX_LATENCY_HISTOGRAM_BUCKET_MS)
+            ? getAppConfig().getLong(MAX_LATENCY_HISTOGRAM_BUCKET_MS)
+            : DEFAULT_MAX_LATENCY_HISTOGRAM_BUCKET_MS;
+    while (curr_duration_ms >= 10) {
```

Comment:
In general, try to keep magic numbers out of code and move them into constants like you did with the default.